adagrams/submit.md

```diff
@@ -83,3 +83,48 @@ The URL to your team pull request: https://github.com/<some-ada-repo>/<project-n
 ##### !end-answer
 ### !end-challenge
 <!-- prettier-ignore-end -->
+
+## Tool Usage Reflection
+
+<!-- prettier-ignore-start -->
+### !challenge
+* type: paragraph
+* id: 0a403185-cde2-4a5b-9bfb-7333ff3db96h
+* title: Tool Usage Reflection
+##### !question
+
+If you used AI tools while working on this project, please share links to the AI chats you used to discuss the project and answer the following questions:
+- How did you confirm that the information the AI shared was up to date and accurate?
+- What feedback did you find useful or think would be helpful to keep in mind for the future?
+- What feedback was not useful or relevant?
```

Comment:
I'm not sure why, but in these instances, "feedback" makes it seem like they ran a completed project through AI to receive feedback. I wonder if something like "suggestions" might be more appropriate.

```suggestion
- What suggestions did you find useful or think would be helpful to keep in mind for the future?
- What suggestions were not useful or relevant?
```

Reply:
That sounds good to move it away from the language we use for a full project review 👍